templates/web-apps/svelte-js/package.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "starter-webapp-sveltekit-js",
+	"version": "1.0.0",
+	"type": "module",
+	"scripts": {
+		"dev": "vite dev",
+		"build": "vite build",
+		"preview": "vite preview",
+		"format": "prettier --write .",
+		"lint": "prettier --check .",
+		"robo:build": "robo build",
+		"robo:deploy": "robo deploy",
+		"robo:dev": "robox dev",
```

Comment:
Robo's `dev` command already handles `vite dev` usage. You can safely remove `vite dev` and `vite build` scripts in favor of `robo dev` and `robo build` and achieve the same result, given that your `vite.mjs` config file has the same contents as `vite.config.js`.

`vite.config.js` technically isn't needed because `/config/vite.mjs` replaces it.